GUIDELINES.md

```diff
@@ -0,0 +1,63 @@
+
+### User Experience
+
+No preset or field is isolated; they are always presented alongside others in various user interfaces that utilize the tagging schema.
+
+- 🔦 **Easy to Pick**: Users must be able to understand and select the correct preset given the limited information available in the user interfaces. Good presets guide the user with clear names and helpful additional documentation `(i)`.
+- 🔎 **Easy to Search**: When searching, similar presets will appear next to each other. Consider and test typical search scenarios. You might need to adjust the names and documentation of other presets to ensure users can make the best decision.
+- 👨‍💻 **Users Are Not Experts**: No prior knowledge of OpenStreetMap or any other background information should be necessary.
+- 🐿️ **Easy Answer**: Users are often on the go and impatient. Fields should allow for quick, straightforward, and clear answers.
+
+### Situational Presets
+
+- 🙈 **Unsearchable Presets**: The tagging schema is not only for adding information but also for presenting existing information. Consider adding an unsearchable preset for tagging that should be highlighted with a preset on the map and with defined fields. Reasons to make a preset unsearchable include: multiple ways to tag something where one method is preferred, or other reasons to hide commonly used tags from the search and list interface to preserve a good [user experience](#user-experience).
+- 🏝️ **Local Presets and Fields**: Generally, presets and fields in OSM should be globally applicable, and efforts should be made to ensure this. However, when local tagging conventions exist or when presets only make sense for certain regions, presets and fields can be given a local filter. This increases the need for thorough testing and makes it more challenging to maintain a good [user experience](#user-experience).
+
+### Tag Updates and Additions
+
+- ➕ **Suggested Additions**: Presets can suggest additional tags. These suggestions must be clearly supported by the wiki and community consensus.
+- 🔄 **Updates**: Deprecation rules can suggest updating tags. Good documentation and consensus are needed for these deprecations.
+
+In both cases, _indicators for consensus_ are:
+- The deprecation is documented in the wiki and is either official (resulting from a proposal process) or long-standing (about a year).
+- There is a significant drop in usage compared to current numbers, with a negative trend ([visible in the graph](https://taghistory.raifer.tech/)).
+- Usage of the deprecated tag remains stagnant for a longer period (about a year).
+
+## 2. Design the Preset
+
+The user interface must be clear, concise, and easy to use, leaving no room for misunderstandings.
+
+- Define the tags required on an object to trigger the preset.
+- Choose a name, category, and define a list of English search terms.
```

Comment:
Another matter of style is that we’re using Title Case in preset and field names: #473.

Reply:
@1ec5 good point. It is just name and alias for Title Case, right? terms are all lowercased?

```
- Choose a name, category, and define a list of American English search terms.
- Use Title Case for the the `name`, `aliases` and `label` property. Use lower case for `terms`. And Title Case or sentences for `strings`-`options`.
```

This is not ideal, yet. We might want to move this to https://github.com/ideditor/schema-builder or to https://github.com/openstreetmap/id-tagging-schema/blob/main/CONTRIBUTING.md or both…

Maybe only mention the most important cases here (name, aliases, terms) and reference one of the other docs?

Reply:
Sure, that sounds fine, as long as it’s mentioned somewhere discoverable.